src/main/java/Duke.java

```diff
@@ -6,5 +9,133 @@ public static void main(String[] args) {
+//                 if (add.contains("unmark")) {
+//                     String[] parts = add.split(" ");
+//                     split = Integer.parseInt(parts[1]);
+//                     markers[split - 1] = " ";
+//                     System.out.println("OK, I've marked this task as not done yet:");
+//                     System.out.println("[ ] " + toDoList[split - 1]);
+//                 } else if (add.contains("mark")) {
+//                    String[] parts = add.split(" ");
+//                    split = Integer.parseInt(parts[1]);
+//                    markers[split-1] = "X";
+//                    System.out.println("Nice! I've marked this task as done:");
+//                    System.out.println("[X] "+ toDoList[split-1]);
+//                }
+//                continue;
+//            } else {
+//                toDoList[counter] = add;
+//                System.out.println("added: " + add);
+//            }
+//            counter++;
+//        }
+
+
+        //level 4
+        // list with 100 rows, 3 columns, 0 - TaskType, 1 - isDone, 2 - TaskName
+        String[][] TaskList = new String[100][3];
+        int ListCounter = 0;
+        for(int row=0; row<100; row++){
+            for(int col=0; col<3; col++){
+                TaskList[row][col] = " ";
+            }
+        }
+        while(true) {
+            Scanner ScannerObject4 = new Scanner(System.in);
+            String input = ScannerObject4.nextLine();
+            if(Objects.equals(input, "bye")){
+                System.out.println("Bye! See you :)");
+                break;
+            } else if (Objects.equals(input, "list")) {
+                //show list
+                System.out.println("Here are the tasks in your list:");
+                for(int task=0; task<ListCounter; task++){
+                    System.out.println((task+1) + ".[" + TaskList[task][0]+"][" + TaskList[task][1]+"] " + TaskList[task][2]);
+                }
+                continue;
+            }
+            String command = input.substring(0, input.indexOf(' '));
+            String content = input.substring(input.indexOf(' ') + 1);
+
+            boolean isToDo = Objects.equals(command, "todo");
+            boolean isDeadline = Objects.equals(command, "deadline");
+            boolean isEvent = Objects.equals(command, "event");
+            boolean isMark = Objects.equals(command, "mark");
+            boolean isUnmark = Objects.equals(command, "unmark");
+
+            if(isToDo){ //add task to list
+                TaskList[ListCounter][0] = "T";
+                TaskList[ListCounter][2] = content;
+                System.out.println("Got it. I've added this task:");
+                System.out.println("[" + TaskList[ListCounter][0]+"][" + TaskList[ListCounter][1]+"] " + TaskList[ListCounter][2]);
+                System.out.println("Now you have "+ (ListCounter+1) +" tasks in your list.");
+            } else if (isDeadline) { // add deadline to list
```

Comment:
As per the Java Coding Standard, ensure that comments are indented relative to their position in the code.